datumaro/components/importer.py

```diff
@@ -100,6 +102,10 @@ def detect(
 
         return FormatDetectionConfidence.LOW
 
+    @classmethod
+    def version(cls, path: str) -> str:
+        return cls.DEFAULT_VERSION
+
```

Comment:
I think you have to implement version checker too not just export/import version meta info.

Reply:
That good idea. I'll prepare a version checker implementation.